src/libostree/ostree-repo-checkout.c

```diff
@@ -217,6 +249,18 @@ create_file_copy_from_input_at (OstreeRepo     *repo,
         return FALSE;
       temp_out = g_unix_output_stream_new (temp_fd, FALSE);
 
+      if (sepolicy_enabled)
+        {
+          g_autofree char *label = NULL;
+          if (!ostree_sepolicy_get_label (options->sepolicy,
+                                          state->selabel_path_buf->str,
+                                          g_file_info_get_attribute_uint32 (file_info, "unix::mode"),
+                                          &label, cancellable, error))
+            return FALSE;
+          if (fsetxattr (temp_fd, "security.selinux", label, strlen (label), 0) < 0)
+            return glnx_throw_errno_prefix (error, "Setting security.selinux");
```

Comment:
We already do that via the
`            return g_prefix_error (error, "Checkout of %s to %s: ", checksum, destination_name), FALSE;`
in the caller.